src/main/java/com/conveyal/datatools/manager/jobs/RecreateBuildImageJob.java

```diff
@@ -63,14 +64,28 @@ public void jobLogic() throws Exception {
                     // See https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/ec2/model/ImageState.html
                     String imageState = image.getState().toLowerCase();
                     if (imageState.equals("pending")) {
+                        if (System.currentTimeMillis() - status.startTime > TimeUnit.HOURS.toMillis(1)) {
+                            terminateInstanceAndFailWithMessage(
+                                "It has taken over an hour for the graph build image to be created! Check the AWS console to see if the image was created successfully."
```

Comment:
This log message is ambiguous. Has the graph completed at this stage?

Reply:
Change the log message to: `The graph image build started over an hour ago. Check in the AWS console whether the image was created successfully.`

Reply:
`graph image build` is not quite accurate. It's recreating the `graph build image` (i.e., the AMI for the graph build instance).